src/main.cpp

```diff
@@ -1621,6 +1621,8 @@ bool GetSpentIndex(CSpentIndexKey &key, CSpentIndexValue &value)
     AssertLockHeld(cs_main);
     if (!fSpentIndex)
         return false;
+    if (mempool.getSpentIndex(key, value))
```

Comment:
Missed this from upstream: https://github.com/zcash-hackworks/zcash-patched-for-explorer/blob/dffb2ebbca44e475171575092db5c54349b3d10c/src/main.cpp#L1634. @garethtdavies noticed this bug and mentioned it here: https://github.com/zcash/zcash/pull/3972#issuecomment-518052328 (thanks, Gareth!) but I thought it's better to fix it in the earliest possible PR.